modules/references/client/components/read-references/ReferenceCounts.js

```diff
@@ -2,6 +2,7 @@ import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
+import { experienceType } from '../../../experiences.prop-types';
```

Comment:
Hm, take the above back — there is some other issue with it. Just not sure why changing import made the app run for a moment:

![image](https://user-images.githubusercontent.com/87168/103292788-2ddadd00-49f7-11eb-81b3-0af451a82f80.png)

Reply:
@simison fixed!